core_brands/db

```diff
@@ -0,0 +1 @@
+../custom_integrations/deutschebahn
```

Comment:
This seems incorrect?

Reply:
Why does it seem incorrect, the pull request in the core adds a virtual integration called db. I thought it wouldn't make any sense to have the db images twice in the repository, therefore I linked them.